app/src/lib/git/diff.ts

```diff
@@ -120,8 +124,10 @@ export async function getCommitDiff(
     args.push(file.status.oldPath)
   }
 
+  const argsToUse = args.filter(arg => arg !== '')
```

Comment:
instead of using a `.filter` here, could we conditionally `.splice` `'-w'` into `args` or something like that? we could make the `whiteSpaceArg` ternary return `[]` or `['-w']`and spread (`...`) that into `args` at creation time.

i'd like to avoid adding arguments that have to be removed later.